.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: npm-publish
+on:
+  workflow_dispatch:
+jobs:
+  npm-publish:
+    name: npm-publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Use Node.js 16.3.0
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.3.0
+          registry-url: "https://registry.npmjs.org"
+      - run: |
+          cd app-services-sdk-js
+          yarn install
+          yarn build
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
```

Comment:
it might be worth trying each release with your own npm / pypi etc. account from this dir structure to ensure the paths are correct

Reply:
after this pr but before we update the openapi generator version